src/datepicker/__tests__/datepicker-keyboard-nav.e2e.js

```diff
@@ -153,7 +153,7 @@ describe('Datepicker - keyboard navigation', () => {
     // check that selected date shows up in the input
     let inputValue = await page.$eval(selectors.input, input => input.value);
     // get rid of any whitespaces in the value to compare
-    expect(inputValue.replace(/\s/g, '')).toBe('2019/03/12–//');
+    expect(inputValue.replace(/\s/g, '')).toBe('2019/03/12');
```

Comment:
I wonder how many integration/unit tests assert on the hyphen and slash symbols and will need to be updated after this change? Is it possible to keep it the same?